src/components/hooks/use-website-status.js

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import { useCookieState } from './use-cookie-state'
+import { BinarySocketBase } from 'common/websocket/socket_base'
+import { getDateFromToday } from 'common/utility'
+
+const WEBSITE_STATUS_COUNTRY_KEY = 'website_status'
+
+export const useWebsiteStatus = () => {
+    const [website_status, setWebsiteStatus] = useCookieState(null, WEBSITE_STATUS_COUNTRY_KEY, {
+        expires: getDateFromToday(7),
```

Comment:
Perhaps we can use a variable the Number.

Reply:
did you mean 7?